config/application.yml

```diff
@@ -0,0 +1,18 @@
+---
+development:
+  config_key:
+  # To develop locally with HTTPS, you will
+  # need to uncomment the two lines below.
+  # Replace <your-local-ip> with the local
+  # network IP address of your development
+  # machine.
+  # When starting the application, set the
+  # HOST variable and run the https command,
+  # like so:
+  # `HOST=0.0.0.0 make run-https`
+  # See the README for more information on
+  # local development.
+  # See also the Procfile at the root of
+  # this repository for configuring mailcatcher.
+  #domain_name: <your-local-ip>:3000
+  #mailer_domain_name: <your-local-ip>:3000
```

Comment:
should we point to a specific part of the README, like "Testing on a mobile device or in a virtual machine", or make those instructions match there?

I guess what I'm really asking is, how can we make this comment shorter and potentially reduce duplication if possible?